helm/reana/Chart.yaml

```diff
@@ -26,7 +26,7 @@ keywords:
   - reusable-science
 type: application
 # Chart version.
-version: 0.7.3
+version: 0.7.4-rc.1
```

Comment:
BTW if we make an "official" -alpha,1 release out of this, you can also update `reana/version.py` and frieds, but let us *not* update `scripts/prefetch-images.sh`, since researchers should stay on 0.7.3 non-alpha...